resources/dicts/patrols/general/border.json

```diff
@@ -182,6 +182,29 @@
     "history_text": [
         "This cat got a scar from a fight with a small dog, but they always say it was a huge dog."
     ]
+},
+{
+    "patrol_id": "gen_bord_abandonedkittypet1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat_injury_kittypet", "nc_sickness",  "respect", "warrior"],
+    "intro_text": "The patrol hears the wailing of a cat and the sound of a monster speeding away.",
+    "decline_text": "It's none of their business really and they have to finish the border patrol.",
+    "chance_of_success": 30,
+    "exp": 15,
+    "success_text": [
+    "They find a well groomed kittypet collapsed by the thunderpath wailing for their twolegs to come back. p_l comforts them and when asked why their twolegs would do this, the only explanation that they can give is that they got sick recently. p_l offers the former kittypet a place in c_n till they are better at least. They tentatively accept whilst looking longely back to the thunderpath."
+    ],
+    "fail_text": [
+    "They discover the cooling body of a kittypet, it seems they attempted to run after the monster with their twolegs in it and were run over by another monster instead. p_l silently sends a prayer for the kittypet and can't help but to curse the twolegs who abandoned such a loyal cat."
```

Comment:
I'm not sure clan cats know that Twolegs are inside monsters? This might need to be reworded, especially since it doesn't seem like they actually saw the kittypet be dropped off.